support/build.gradle

```diff
@@ -4,6 +4,8 @@ dependencies {
     api project(':management')
     api project(':yubiotp')
 
+    api 'org.slf4j:slf4j-api:2.0.7'
```

Comment:
I think this can be an `implementation` dependency? I didn't see it in any public members or method signatures, but I could have missed some of course.